src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_Helpers.cs

```diff
@@ -129,63 +112,28 @@ private static bool IsAccessible(ISymbol symbol)
             if (equivalentTypesWithDifferingAssemblies.Count > 0)
             {
                 // Step 3a) Ensure that all pairs of named types in equivalentTypesWithDifferingAssemblies are indeed equivalent types.
-                return VerifyForwardedTypes(
-                    equivalentTypesWithDifferingAssemblies, searchSymbol, symbolToMatch,
-                    solution, searchSymbolCompilation, symbolToMatchCompilation);
+                return VerifyForwardedTypes(solution, equivalentTypesWithDifferingAssemblies, cancellationToken);
             }
 
             // 3b) If no such named type pairs were encountered, symbols ARE equivalent.
             return true;
         }
 
         private static bool NamespaceSymbolsMatch(
+            Solution solution,
             INamespaceSymbol namespace1,
             INamespaceSymbol namespace2,
-            Solution solution)
+            CancellationToken cancellationToken)
         {
-            return OriginalSymbolsMatch(namespace1, namespace2, solution);
+            return OriginalSymbolsMatch(solution, namespace1, namespace2, cancellationToken);
         }
 
         // Verifies that all pairs of named types in equivalentTypesWithDifferingAssemblies are equivalent forwarded types.
```

Comment:
If you're having to make a code change in this file due to feedback switch this to a XML doc comment. Not worth doing by itself.

Reply:
done.